invenio_base/app.py

```diff
@@ -194,7 +194,7 @@ def blueprint_loader(app, entry_points=None, modules=None):
 
 
 def converter_loader(app, entry_points=None, modules=None):
-    """Default converter loader.
+    """Return default converter loader.
```

Comment:
Define sounds better, although I am not really big fan of D401 as it's defined in pydocstyle 2.0.